extensions/azurePublishNew/src/node/availableResources.ts

```diff
@@ -32,6 +32,7 @@ export const getProvisionServices = (config: ProvisionConfig): Record<string, Re
   return {
     appRegistration: getAppRegistrationProvisionService(config),
     webApp: getWebAppProvisionService(config),
+    servicePlan: getAppServiceProvisionService(config),
```

Comment:
FYI. With an associative array, the order isn't guaranteed to be preserved.

Reply:
right! I was just organizing